lib/split/experiment.rb

```diff
@@ -394,93 +392,109 @@ def calc_time
 
     def jstring(goal = nil)
       js_id = if goal.nil?
-                name
-              else
-                name + "-" + goal
-              end
-      js_id.gsub('/', '--')
+        name
+      else
+        name + "-" + goal
+      end
+      js_id.gsub("/", "--")
     end
 
-    protected
-
-    def experiment_config_key
-      "experiment_configurations/#{@name}"
+    def cohorting_disabled?
+      @cohorting_disabled ||= begin
+        value = redis.hget(experiment_config_key, :cohorting)
+        value.nil? ? false : value.downcase == "true"
+      end
     end
 
-    def load_metadata_from_configuration
-      Split.configuration.experiment_for(@name)[:metadata]
+    def disable_cohorting
+      @cohorting_disabled = true
+      redis.hset(experiment_config_key, :cohorting, true)
     end
 
-    def load_metadata_from_redis
-      meta = redis.get(metadata_key)
-      JSON.parse(meta) unless meta.nil?
+    def enable_cohorting
+      @cohorting_disabled = false
+      redis.hset(experiment_config_key, :cohorting, false)
     end
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/experiment.rb#L409-L424